app/css/components/tooltips/concept.css

```diff
@@ -91,4 +93,25 @@
             @apply text-darkGray leading-paragraph font-medium;
         }
     }
+
+    &::before {
+        content: "";
+        width: 100%;
+        height: 8px;
+        left: 0;
+        top: 0;
+        transform: translateY(-100%);
+        background: rgba(0, 0, 0, 0);
```

Comment:
```suggestion
        background: transparent;
```

![Showing the colour picker for the transparent keyword colour. It matches #00000000, which is rbga(0, 0, 0, 0)](https://user-images.githubusercontent.com/1964376/99968089-fa25eb00-2d98-11eb-8e7a-f73fa5318858.png)

It's usually easier to parse with intent.